src/assets/scripts/base/PositionModel.js

```diff
@@ -1,21 +1,38 @@
 import _uniqueId from 'lodash/uniqueId';
-import { sin, cos } from '../math/core';
+import {
+    calculateDistanceToPointForX,
+    calculateDistanceToPointForY,
+    adjustForMagneticNorth,
+    hasCardinalDirectionInCoordinate
+} from './positionModelHelpers';
+import { distanceToPoint } from '../math/circle';
 import {
     degreesToRadians,
     parseCoordinate,
     parseElevation
 } from '../utilities/unitConverters';
 
 /**
- * @property REGEX
- * @type {Object}
+ * @property LATITUDE_INDEX
+ * @type {number}
  * @final
```

Comment:
that it shouldn't ever be modified.  It essentially means that it's a constant.  with es6, the build would fail if you tried to modify a constant, the `@final` is just a little extra doc tag.